src/dbnode/storage/index_test.go

```diff
@@ -612,6 +756,31 @@ func verifyFlushForShards(
 	require.Equal(t, expectedDocs, actualDocs)
 }
 
+func readIndexInfoFileResult(
```

Comment:
Nit: please think of a better name, eg. `newReadIndexInfoFileResult` or smth.
The current name incorrectly implies that this function is reading something.